app/src/main/java/ch/epfl/skysync/screens/admin/AdminFlightDetailsScreen.kt

```diff
@@ -150,7 +150,7 @@ fun FlightDetailBottom(
                   .testTag("ConfirmButton")) {
             TextButton(
                 onClick = { confirmClick() },
-                shape = RoundedCornerShape(topEnd = 8.dp, bottomEnd = 8.dp),
+                shape = RoundedCornerShape(topEnd = 16.dp, bottomEnd = 16.dp),
```

Comment:
maybe that kind of parameters (rounded shape of button) could be unified in a style sheet (like we did with the colors)